packages/material-ui/src/CardMedia/CardMedia.js

```diff
@@ -15,6 +15,9 @@ export const styles = {
   /* Styles applied to the root element if `component="video, audio, picture, iframe, or img"`. */
   media: {
     width: '100%',
+    height: '100%', 
```

Comment:
I'm not sure we need this anymore. Can you try to remove it?

Reply:
If we remove the height the image won't fill from the center in the div. 

Try commenting out line 35 here: https://codesandbox.io/s/91yp7jvoxw. 

I feel like when an image is loaded into a div you want it to fill from the center of the image, not the top, which is what happens if you remove the height. I think usually the center of an image is more important and should be the focal point when making the component responsive. 

But if you think otherwise, I can remove it no problem.

Reply:
I'm removing the height, the visual regression tests have captured a change. We encourage people to rely on the img height property.

Reply:
Let's follow the documentation demos.